frb_codegen/src/config.rs

```diff
@@ -12,6 +12,9 @@ use toml::Value;
 #[derive(StructOpt, Debug, PartialEq, Deserialize)]
 #[structopt(setting(AppSettings::DeriveDisplayOrder))]
 pub struct RawOpts {
+    /// Path of input cargo.toml file
+    #[structopt(short, long)]
+    pub cargo_path: String,
```

Comment:
You might have to remove the short flag here since it's conflicting with c_output.